merlin/main.py

```diff
@@ -560,8 +535,7 @@ def setup_argparse() -> None:
         action="store",
         type=str,
         default=None,
-        help="Specify a path to write the workflow to. Defaults to current "
-        "working directory",
+        help="Specify a path to write the workflow to. Defaults to current " "working directory",
```

Comment:
```suggestion
        help="Specify a path to write the workflow to. Defaults to current working directory"
```

Reply:
Also linters (like Black) treat trailing commas as "I plan to extend this (someday)" which also implicitly means "do not reformat this, it will grow", so some people use that to prevent a reformat rather than explicitly markup "do not lint".